<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visiut3</class>
 <widget class="QMainWindow" name="Visiut3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisIUT 3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>192</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="0">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::NoDrag</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="ActionNew"/>
    <addaction name="ActionOpen"/>
    <addaction name="ActionSave"/>
    <addaction name="ActionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuFiltres">
    <property name="title">
     <string>Filtres</string>
    </property>
    <addaction name="ActionNegative"/>
    <addaction name="ActionMirror"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="ActionCancel"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuEdition"/>
   <addaction name="menuFiltres"/>
   <addaction name="menuOutils"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionNew"/>
   <addaction name="ActionOpen"/>
   <addaction name="ActionSave"/>
   <addaction name="separator"/>
   <addaction name="ActionNegative"/>
   <addaction name="ActionMirror"/>
   <addaction name="separator"/>
   <addaction name="ActionCancel"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <action name="ActionNew">
   <property name="icon">
    <iconset>
     <normalon>Icone/page_white_add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="ActionOpen">
   <property name="icon">
    <iconset>
     <normalon>Icone/folder_add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="iconText">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActionQuitter">
   <property name="icon">
    <iconset>
     <normalon>Icone/door_in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ActionMirror">
   <property name="icon">
    <iconset>
     <disabledoff>Icone/shading.png</disabledoff>
    </iconset>
   </property>
   <property name="text">
    <string>Mirroir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionPropriete">
   <property name="icon">
    <iconset>
     <normalon>Icone/help.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Proprietes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="ActionNegative">
   <property name="icon">
    <iconset>
     <normalon>Icone/color_wheel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Negatif</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="ActionSave">
   <property name="icon">
    <iconset>
     <normalon>Icone/bullet_disk.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>Icone/heart.png</normaloff>Icone/heart.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionEffacer">
   <property name="icon">
    <iconset>
     <normaloff>Icone/cancel.png</normaloff>Icone/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
   <property name="toolTip">
    <string>Effacer l'image courante</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSuppression">
   <property name="icon">
    <iconset>
     <normaloff>Icone/cross.png</normaloff>Icone/cross.png</iconset>
   </property>
   <property name="text">
    <string>Suppression d√©finitive</string>
   </property>
  </action>
  <action name="ActionCancel">
   <property name="icon">
    <iconset>
     <normaloff>Icone/arrow_turn_left.png</normaloff>Icone/arrow_turn_left.png</iconset>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ActionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>Visiut3</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
